MetaMask/metamask-extension
# Feat/collectibles the return

Explanation:  Add required wiring + front-end work to get viewing/adding NFTs in extension

In designs but excluded here
- Send flow (will be included in fast-follow PR)
- `Report as Scam` flow (the functionality here is not sufficiently spec'd out to implement)

Screenshots:
<img width="684" alt="Screen Shot 2021-11-30 at 4 09 50 PM" src="https://user-images.githubusercontent.com/34557516/144136219-ae84e63d-822e-4a94-98c6-420eac365a38.png">
<img width="944" alt="Screen Shot 2021-11-30 at 4 10 54 PM" src="https://user-images.githubusercontent.com/34557516/144136279-aa553a70-ebf9-48b6-a445-b2877e947a93.png">

Manual testing steps:  
  - Purchase or mint or have an (or preferably many) NFT(s) 
  
  Detection:
  - (only available on Mainnet)
  - Turn on `OpenSea Api` & `NFT autodetection` in settings -> experimental
  - Check that your NFT has been added to the NFTs tab
  
  Manual:
  - Navigate to the NFTs tab
  - Click "Import NFTs"
  - Enter `address`/`tokenId` in `/add-collectible` form
  
1 (with NFT you own)
  - Check that your NFT has been successfully added (should see green success toast)
 
2 (with NFT you don't own)
 - Check that the NFT has not been added (should see yellow warning toast: "collectible not owned by user")
